test/runtests.jl

```diff
@@ -0,0 +1,9 @@
+using WinRPM
+using Test
+
+if haskey(ENV, "CI") && haskey(ENV, "WINRPM_DO_THE_TEST") && ENV["WINRPM_DO_THE_TEST"] == "true"
```

Comment:
this env var would need to be set in the appveyor config for this to do anything

Reply:
that should be set in the yaml where it's visible and can be modified by people who don't have access to that settings page - generally the UI should be avoided, the yaml takes precedence for anything set both places